gtfs_realtime_translators/translators/path_rail.py

```diff
@@ -0,0 +1,137 @@
+        '861_33S_23S': '781738',
+        '861_33S_33S': '781740',
+        '861_JSQ_JSQ': '781724',
+        '861_JSQ_GRV': '781727',
+        '861_JSQ_NEW': '781729',
+        '861_JSQ_CHR': '781733',
+        '861_JSQ_09S': '781735',
+        '861_JSQ_14S': '781737',
+        '861_JSQ_23S': '781739',
+        '861_JSQ_33S': '781741',
+        '862_NWK_NWK': '781719',
+        '862_NWK_HAR': '781721',
+        '862_EXP_HAR': '781721',
+        '862_NWK_JSQ': '781725',
+        '862_EXP_JSQ': '781725',
+        '862_NWK_GRV': '781727',
+        '862_EXP_GRV': '781727',
+        '862_NWK_EXP': '781731',
+        '862_EXP_EXP': '781731',
+        '862_NWK_WTC': '794724',
+        '862_WTC_NWK': '781718',
+        '862_EXP_NWK': '781718',
+        '862_WTC_HAR': '781720',
+        '862_WTC_JSQ': '781722',
+        '862_WTC_GRV': '781726',
+        '862_WTC_EXP': '781730',
+        '862_WTC_WTC': '794724',
+        '1024_33S_HOB': '781715',
+        '1024_33S_JSQ': '781723',
+        '1024_33S_GRV': '781726',
+        '1024_33S_NEW': '781728',
+        '1024_33S_CHR': '781732',
+        '1024_33S_09S': '781734',
+        '1024_33S_14S': '781736',
+        '1024_33S_23S': '781738',
+        '1024_33S_33S': '781740',
+        '1024_JSQ_HOB': '781716',
+        '1024_JSQ_JSQ': '781724',
+        '1024_JSQ_GRV': '781727',
+        '1024_JSQ_NEW': '781729',
+        '1024_JSQ_CHR': '781733',
+        '1024_JSQ_09S': '781735',
+        '1024_JSQ_14S': '781737',
+        '1024_JSQ_23S': '781739',
+        '1024_JSQ_33S': '781741'
+    }
+
+    """
+       Since PATH GTFS data have stops that are, in most cases, unique to a route, direction, service date, and/or 
+       destination, a mapping is created to ensure we return the appropriate stop_id for a station's arrival updates.
+       
+       Keys are based off the line color (a hex value that can be mapped to a GTFS route_id), the destination, and the 
+       current station for which the request has been made.
+    """
+
+    def __call__(self, data):
+        json_data = json.loads(data)
+        entities = self.__make_trip_updates(json_data)
+        return FeedMessage.create(entities=entities)
+
+    @classmethod
+    def __make_trip_updates(cls, data):
+        trip_updates = []
+        now = int(pendulum.now().timestamp())
+
+        arrivals = data['results']
+        for arrival in arrivals:
+            arrival_updates = arrival['messages']
+            for idx, update in enumerate(arrival_updates):
+                try:
+                    route_id = cls.ROUTE_ID_LOOKUP[update['lineColor']]
+                    destination = arrival['target']
+                    station = arrival['consideredStation']
+
+                    key = route_id + '_' + destination + '_' + station
+                    stop_id = cls.STOP_ID_LOOKUP[key]
+                    arrival_time = now + math.floor(update['secondsToArrival'] / 60) * 60
+                    custom_status = update['arrivalTimeMessage']
```

Comment:
Can we leave this field blank? It isn't intended to hold info about arrival times and we already have that data in the `arrival_time` field.

Reply:
Good call. It's not needed, so I removed it.